types/Modifiers.d.ts

```diff
@@ -1,11 +1,6 @@
 export interface RangeModifier {
-  from: Date;
-  to: Date;
-}
-
-export interface RangeModifier {
-  from: Date;
-  to: Date;
+  from: Date | undefined;
+  to: Date | undefined;
```

Comment:
If you want set undefined to type property better to use `?` if it's possible.
```
export interface RangeModifier {
  from?: Date;
  to?: Date;
}
```
Nevertheless, I think there should be 
```
export interface RangeModifier {
  from: Date | null;
  to: Date | null;
}
```
Because In realisation of the function `addDayToRange` it has fallback to nulls.
```
export function addDayToRange(day, range = { from: null, to: null }) {}
```